### ISSUE TITLE
Loss does not converge

### ISSUE THREAD

**Issue by ZhuoranLyu** [thomasschmied/Speech_Recognition_with_Tensorflow#2]
I use only the first 300 texts and audios to train the model. However, the loss went down to about 1.7 and then it does not converge any more. Any help would be appreciated.

**Comment by AzizCode92** [comment]
I have also a score of 1.7 after 257 epochs out of 500.
I think if you modify the early stop criteria in the code you may tolerate it to more than "5 epochs without improvement" as mentioned in the code.

**Comment by thomasschmied** [comment]
Yes, I met similar difficulties during training - it can be tricky. Try restoring your half-trained model and use different, higher, learning rates!

**Comment by ZhuoranLyu** [comment]
I use the first 1000 txts and the loss converges to about 0.3. However, if I use the first 2500 txts, the loss went down to about 1.0 and it never went down any more. No matter how I set the learning rate.

**Comment by AzizCode92** [comment]
@ZhuoranLyu  I also used the 1000 first txts and the loss converged to 1.7.
Did you modify something in the configuration of the model?
If yes, can you share it with us?

**Comment by ZhuoranLyu** [comment]
@AzizCode92 Nothing changed. I tried to change the num_layers and the hidden_size, but neither works....

**Comment by AzizCode92** [comment]
I would like to know if those parameters  in the training are the values used by the state of the art of listen-attend-and-spell paper or just custom choice?
I didn't see anything in the experiments section of the paper related to the training parameters.
I think that's an important point that we can't neglect if we want to reproduce their baseline results.
Thanks

**Comment by ZhuoranLyu** [comment]
The training parameters are described in the original paper.
